front/src/stats/Courses/Explorer.jsx

```diff
@@ -166,24 +166,24 @@ const Explorer = (props) => {
             <Col id="functionalities">
               <h4>Consultar Analítica</h4>
               <ul className="list-group-eol">
-                {/* <li>
+                <li>
                   <span className="toggle-arrow">
                     <FontAwesomeIcon icon={faChevronRight} />
                   </span>
-                  <Link to={`/courses/${key}/times/${start}/${end}`}>
+                  <Link to={`/courses/${key}`}>
                     Ver estadísticas generales{' '}
                     <FontAwesomeIcon
                       icon={faExternalLinkAlt}
                       className="float-right"
                     />
                   </Link>
-                </li> */}
-                <li>
+                </li>
+                {/* <li>
                   <span className="toggle-arrow">
```

Comment:
Make all the views accessible unless the charts bug in-between view changes

Reply:
Bug in-between view is related to useMemo